molecule/model/schema_v2.py

```diff
@@ -944,6 +945,20 @@ class Validator(cerberus.Validator):
     def __init__(self, *args, **kwargs):
         super(Validator, self).__init__(*args, **kwargs)
 
+    def _validate_unique(self, unique, field, value):
+        """Ensure value uniqueness.
+
+        The rule's arguments are validated against this schema:
+        {'type': 'boolean'}
+        """
+        if unique:
+            root_key = self.schema_path[0]
```

Comment:
See http://docs.python-cerberus.org/en/stable/customize.html#validator-document-path-validator-schema-path.

Seems fine to index since it will be populated with the path to the value.